internal/db/rdb/mysql/dbr/connection.go

```diff
@@ -14,6 +14,8 @@
 // limitations under the License.
 //
 
+// Package dbr provides the dbr functions for handling with databases.
```

Comment:
please create a file `doc.go` and write the package there.

Reply:
Why doc.go is needed in this case 🤔 ?
I'm sorry, but I'm not sure.